### ISSUE TITLE
omp_set_nested routine deprecated

### ISSUE THREAD

**Issue by dtejadam** [yuqiyuqitan/SPACEc#57]
Hi, 
I am in the step of assign cell types by Clustering, please see the following error: 

# clustering
adata = sp.tl.clustering(
    adata, 
    clustering='leiden', # can choose between leiden and louvian
    n_neighbors=10, # number of neighbors for the knn graph
    resolution = 1, #clustering resolution (higher resolution gives more clusters)
    reclustering = False, # if true, no computing the neighbors
    marker_list = marker_list, #if it is None, all variable names are used for clustering
    seed=clustering_random_seed, # random seed for clustering - reproducibility
)
2024-10-04 12:04:51.188460: I tensorflow/core/platform/cpu_feature_guard.cc:193] This TensorFlow binary is optimized with oneAPI Deep Neural Network Library (oneDNN) to use the following CPU instructions in performance-critical operations:  SSE4.1 SSE4.2
To enable them in other operations, rebuild TensorFlow with the appropriate compiler flags.
INFO:root: * TissUUmaps version: 3.1.1.6
Computing neighbors and UMAP
OMP: Info #276: omp_set_nested routine deprecated, please use omp_set_max_active_levels instead.

Kernel died, restarting

- Additional question: how did you trained the Annotated tonsil data to be used as reference using STELLAR or SVM ? 

Thanks !
